etcdctl/ctlv3/command/global.go

```diff
@@ -114,6 +114,9 @@ func clientConfigFromCmd(cmd *cobra.Command) *clientConfig {
 		fs.VisitAll(func(f *pflag.Flag) {
 			fmt.Fprintf(os.Stderr, "%s=%v\n", flags.FlagToEnv("ETCDCTL", f.Name), f.Value)
 		})
+	} else {
+		clientv3.SetLogger(grpclog.NewLoggerV2(ioutil.Discard, ioutil.Discard, ioutil.Discard))
+		grpclog.SetLoggerV2(grpclog.NewLoggerV2(ioutil.Discard, ioutil.Discard, ioutil.Discard))
```

Comment:
why setting clientv3 logger is not good enough?

Reply:
`clientv3.SetLogger` is for clientv3 logs (e.g. balancer marked as unhealthy...)
`grpclog.SetLoggerV2` is for gRPC's internal logs (e.g. sub connection changes...)

Reply:
i think clientv3's logger should control gRPC actually. it should mute gRPC since from users perceptive gRPC is an implementation detail of the client.